src/middleware.ts

```diff
@@ -1,28 +0,0 @@
-import { NextRequest, NextResponse } from 'next/server';
-import { logResponse } from '@/utils/loggerUtils';
-
-
-export const FHIR_AUTHORIZATION_TOKEN = "fhir-authorization-token";
-
-export const middleware = (request: NextRequest): NextResponse => {
```

Comment:
Jeg ser at du har fjernet middleware til fordel for en hjemmelaget proxy, som forsåvidt er greit. Men NextJs har innebygd støtte for proxy gjennom [rewrites](https://nextjs.org/docs/app/api-reference/next-config-js/rewrites). Det hadde vært fint vi kunne forholdt oss til det som er støttet av NextJs og som er dokumentert.

Du har også utelatt request/response logging som tidligere var støttet.

Reply:
Ser ikkje heilt at rewrites kan støtte det vi gjere her, er ikkje det berre ein statisk proxy som ikkje forandre på request? Vi må jo kunne legge på header på request for å dekke vårt behov.

Har endra request/response logginga i proxy handleren til å bruke logRequest og logResponse funksjonane no 👍

Reply:
Du har rett. Den vil nok ikke løse behovet vårt dynamisk. 
Flott at loggingen er tilbake 👍